atest/resources/html/javascript/click.html

```diff
@@ -16,6 +22,8 @@ <h1>Clicking</h1>
     <p id="output">initial output</p>
     <button type="button" id="singleClickButton" onclick="singleClick()">Click me!</button>
     <button type="button" id="doubleClickButton" ondblclick="doubleClick()">Double click me!</button>
+    <button type="button" id="twoButtons" onclick="twoButtons1()">Button 1</button>
+    <button type="button" id="twoButtons" onclick="twoButtons2()">Button 2</button>
```

Comment:
Yeah, using same id multiple times is invalid and how browsers handle it may differ. In these tests it probably would be better to avoid them and use, for example, same class with different elements.